scalar_test.go

```diff
@@ -0,0 +1,84 @@
+package opslevel_test
+
+import (
+	"encoding/json"
+	"github.com/hasura/go-graphql-client"
+	"github.com/opslevel/opslevel-go/v2023"
+	"github.com/rocktavious/autopilot/v2022"
+	"testing"
+)
+
+type TestInput struct {
+	Key1 opslevel.ID  `json:"key1"`
```

Comment:
Is there a reason these are `opslevel.ID` instead of `interface{}` like the update struct?

Reply:
The updated struct was a change from your branch that I was based on when i made this change.  Ideally if we are good with this we'll use `opslevel.ID` everywhere from here on out